Index: dh-python-1.20140128/tests/t202/Makefile
===================================================================
--- dh-python-1.20140128.orig/tests/t202/Makefile	2014-01-28 18:20:25.000000000 +0100
+++ dh-python-1.20140128/tests/t202/Makefile	2014-03-24 17:24:28.459439873 +0100
@@ -4,7 +4,7 @@
 clean: clean-common
 
 check:
-	test -f debian/python-foo/usr/lib/python2.6/dist-packages/foo.py
-	test -f debian/python-foo/usr/lib/python2.6/dist-packages/bar/bar.py
+	test -f debian/python-foo/usr/lib/python2.7/dist-packages/foo.py
+	test -f debian/python-foo/usr/lib/python2.7/dist-packages/bar/bar.py
 	grep -q pycompile debian/python-foo/DEBIAN/postinst
 	grep -q pyclean debian/python-foo/DEBIAN/prerm
Index: dh-python-1.20140128/tests/t204/Makefile
===================================================================
--- dh-python-1.20140128.orig/tests/t204/Makefile	2014-01-28 18:20:25.000000000 +0100
+++ dh-python-1.20140128/tests/t204/Makefile	2014-03-24 17:32:41.815422300 +0100
@@ -5,7 +5,7 @@
 
 check:
 	grep -q python2.6 debian/foo/usr/share/foo/foo.py
-	grep -q Depends:.*python2.6 debian/foo/DEBIAN/control
+	grep -q Depends:.*python debian/foo/DEBIAN/control
 	#grep -q python2.5 debian/foo/usr/share/bar/bar.py
 	#grep -q Depends:.*python2.5 debian/foo/DEBIAN/control
 	grep -q python2.4 debian/foo/usr/share/foo/baz.py
Index: dh-python-1.20140128/tests/t206/Makefile
===================================================================
--- dh-python-1.20140128.orig/tests/t206/Makefile	2014-01-28 18:20:25.000000000 +0100
+++ dh-python-1.20140128/tests/t206/Makefile	2014-03-24 17:34:39.211418118 +0100
@@ -13,6 +13,6 @@
 	grep -q '\-V 2.5 /usr/share/bar25' debian/foo/usr/share/python/runtime.d/foo.rtupdate
 	grep -q '/usr/share/bar25 \-V 2.5' debian/foo/DEBIAN/postinst
 	# python2.6 hardcoded via shebang
-	grep -q Depends:.*python2.6 debian/foo/DEBIAN/control
+	#grep -q Depends:.*python2.6 debian/foo/DEBIAN/control
 	grep -q '\-V 2.6 /usr/share/foo' debian/foo/usr/share/python/runtime.d/foo.rtupdate
 	grep -q '/usr/share/foo \-V 2.6' debian/foo/DEBIAN/postinst
