Index: b/dhpython/_defaults.py
===================================================================
--- a/dhpython/_defaults.py
+++ b/dhpython/_defaults.py
@@ -27,11 +27,11 @@
 
 SUPPORTED = {
     'cpython2': [(2, 7)],
-    'cpython3': [(3, 3)],
+    'cpython3': [(3, 4)],
     'pypy': [(2, 0)]}
 DEFAULT = {
     'cpython2': (2, 7),
-    'cpython3': (3, 3),
+    'cpython3': (3, 4),
     'pypy': (2, 0)}
 
 log = logging.getLogger('dhpython')
Index: b/tests/Makefile
===================================================================
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -3,9 +3,9 @@
 # hardcode default/supported Python versions to make tests more reliable
 # Note that all these packages have to be installed during tests
 export DEBPYTHON_DEFAULT=2.7
-export DEBPYTHON_SUPPORTED=2.6,2.7
-export DEBPYTHON3_DEFAULT=3.3
-export DEBPYTHON3_SUPPORTED=3.2,3.3
+export DEBPYTHON_SUPPORTED=2.7
+export DEBPYTHON3_DEFAULT=3.4
+export DEBPYTHON3_SUPPORTED=3.4
 
 # enable or disable tests here:
 TESTS := test101 test201 test202 test203 test204 test205 test206 test207 test301 test302 test303 test304
